### DISCUSSION TITLE
How to compile for windows

### DISCUSSION THREAD

**Discussion by TheCodeHermit** [ruffle-rs/ruffle#15225] [Q&A]
The build instructions only have instructions for linux and I am really struggling to cross compile an exe. I know it is possible because of the releases for windows

**Comment by adrian17** [comment]
AFAIK, the only real required components are the Rust compiler (if you didn't install it before, it'll also need you to install the Visual Studio toolchain it requires - the instructions are shown when installing it with Rustup) and Java.

With these two, a standard `cargo run (--release)` should be enough to launch the desktop player on Windows. That said, not sure what you mean by "cross compile".
